charts/prometheus-statsd-exporter/templates/deployment.yaml

```
@@ -14,6 +14,7 @@ spec:
       {{- include "prometheus-statsd-exporter.selectorLabels" . | nindent 6 }}
   template:
     metadata:
+      checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
       {{- with .Values.podAnnotations }}
       annotations:
```

`checksum/config` is an annotation:

```suggestion
      annotations:
        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
        {{- with .Values.podAnnotations }}
```

Please, raise the chart version in `Chart.yaml`.